content/docs/command-reference/import-url.md

```diff
@@ -88,6 +88,12 @@ Specific explanations:
   running. DVC automatically expands this URL into a regular S3, SSH, GS, etc
   URL by appending `/path/to/file` to the `myremote`'s configured base path.
 
+The `deps` field in the generated `.dvc` file contains a hash value along with
+the `path` to track changes in the remote file/directory. The hash type depends
+on the type of remote location (protocol) being tracked. See
+[`dvc.lock` file](/doc/user-guide/dvc-files-and-directories#dvclock-file) for
+info on hash types for specific remote location types.
+
```

Comment:
Yeah this whole paragraph is probably unnecessary but thanks. The ref. is already too long TBH. Users can find out about .dvc file specs in its dedicated guide, that's not the main purpose for a command ref.

Reply:
Okay, so where should I link `dvc.lock file` ?

Reply:
Why did you decide to edit this command reference if there's no existing places that relate to dvc.lock? I'm going to just merge this with the few typo fixed but I'll have to uncheck that box in the original index so this can be re-evaluated.

Reply:
I actually thought that we needed to add some reference to the hash values in each file or describe where it was needed based on the original issue. But does this mean that there is no place we can add a reference to `dvc.lock` or provide info about different types of file hashes in `import-url`?

Reply:
> does this mean that there is no place we can add a reference to dvc.lock or provide info about different types of file hashes in import-url?

Idk, can you answer that question? 🙂 Let's go back to analyzing the checkboxes of #1469 and please comment there if any one seems unaplicable.